packages/config/src/projects/layer2s/sxnetwork.ts

```diff
@@ -52,21 +52,5 @@ export const sxnetwork: Layer2 = orbitStackL2({
     adjustCount: { type: 'SubtractOne' },
     startBlock: 1,
   },
-  nonTemplatePermissions: [
```

Comment:
Just removed?

Reply:
Yes, because we disco drive this project. If you disco drive we assume that you can't say "yes, but also this". If you want something more you need to add it in the discovery, not in the .ts file.